oneclick.py

```diff
@@ -957,7 +961,21 @@ class OneClickImportMonitor(OneClickSyncMonitor):
     SERVICE_NAME = "OneClick Full Import"
     
     def invoke(self):
-        return self.api.populate_all_catalog()
+        timestamp = self.timestamp()
+        if timestamp.counter and timestamp.counter > 0:
+            self.log.debug(
+                "Collection %s has already had its initial import; doing nothing.", 
+                self.collection.name or self.collection.id
+            )
+            return 0, 0
+        result = self.api.populate_all_catalog()
+
+        # Record the work was done so it's not done again.
+        if not timestamp.counter:
+            timestamp.counter = 1
+        else:
+            timestamp.counter += 1
+        return result
```

Comment:
Here's the change that stops the OneClickImportMonitor from doing work more than once for a given collection. I toyed with adding a `--force` option but it was way too much work and we don't need it ATM.